plugins/janus_sip.c

```diff
@@ -1973,7 +2024,7 @@ static void janus_sip_hangup_media_internal(janus_plugin_session *handle) {
 		session->media.autoaccept_reinvites = TRUE;
 		session->media.ready = FALSE;
 		session->media.on_hold = FALSE;
-		session->status = janus_sip_call_status_closing;
+		update_session_status(session, janus_sip_call_status_closing);
```

Comment:
Can you rename this function to `janus_sip_call_update_status`? It makes it more consistent with the code style of other calls.